src/ScriptVerification/HelloWorld.agda

```diff
@@ -150,7 +150,8 @@ opaque
   succeedExample : ComputationResult String UTxOState
   succeedExample = UTXO-step initEnv ⟦ initState , 0 , ∅ , 0 ⟧ᵘ  succeedTx
 
-  _ : isSuccess succeedExample ≡ true
+  --  This is now false since `minUTxOValue` was replaced with `coinsPerUTxOByte` (= 4310 in `ScriptVerification.Lib`)
```

Comment:
I think it' a bad idea to break these tests. It'd be better to set `coinsPerUTxOByte = 0` with a note to adjust that there.